### ISSUE TITLE
Release New Version

### ISSUE THREAD

**Issue by ElfoLiNk** [rkcpi/logback-gelf-appender#15]
Hello,
can you please release a new version with updated gelfclient?

The latest released version has this two vulnerability:
 
```
✗ Medium severity vulnerability found on com.fasterxml.jackson.core:jackson-core@2.8.4
- desc: Denial of Service (DoS)
- from: de.appelgriepsch.logback:logback-gelf-appender@1.4-SNAPSHOT > org.graylog2:gelfclient@1.4.1 > com.fasterxml.jackson.core:jackson-core@2.8.4

✗ Medium severity vulnerability found on io.netty:netty-handler@4.1.6.Final
- desc: Insecure Defaults
- from: de.appelgriepsch.logback:logback-gelf-appender@1.4-SNAPSHOT > org.graylog2:gelfclient@1.4.1 > io.netty:netty-handler@4.1.6.Final
```

Thank you.
Best Regards,
Matteo

**Comment by rkcpi** [comment]
Hi,

you performed your checks with outdated dependencies. This project uses the newest version of org.graylog2:gelfclient which uses updated dependencies of jackson-core and netty-handler.

Closing this.

**Comment by ElfoLiNk** [comment]
Yes master is using the updated versions but you didn't release a new version https://github.com/rkcpi/logback-gelf-appender/releases. This issue was to release a new logback-gelf-appender version

**Comment by rkcpi** [comment]
You're right of course! Will release a new version after #16 is confirmed resolved.

**Comment by rkcpi** [comment]
I just released version 1.4: https://github.com/rkcpi/logback-gelf-appender/releases/tag/logback-gelf-appender-1.4

Should be up on Maven Central in a few minutes.
